spec/models/mets_document/mods_document_spec.rb

```diff
@@ -200,7 +242,7 @@
     let(:mets_file) { Rails.root.join("spec", "fixtures", "mets", "pudl0038-7350.mets") }
 
     it "accesses the shelfLocator field" do
-      expect(mods_document.shelf_locator).to contain_exactly "Box AD01, Item 7350"
+      expect(mods_document.shelf_locator).to contain_exactly "Mudd, Box AD01, Item 7350"
```

Comment:
Could we run this by Alexis? It seems redundant to me with the holding location.

Reply:
It might be redundant, but sometimes that holding location isn't "Mudd", and it's like, "Curator's Office"

Reply:
```
Trey Pendragon [11:41 AM]
@Alexis A. I just added a change for ingest of Russian movie posters which displays the location as "Curator's Office, Cabinet 11/06". Unfortunately the MODS in PUDL also has a subLocation for Mudd so things like grounds and buildings would show the location as "Mudd, MC085. Box 135." Is that alright?


Alexis A. [11:42 AM]
@Trey Pendragon i don't see any problem with that
```